code/modules/power/singularity/particle_accelerator/particle.dm

```diff
@@ -27,32 +29,39 @@
 	source = null
 	movetotarget = 1
 
+
 /obj/effect/accelerated_particle/weak
 	movement_range = 8
-	energy = 5
+	energy = PARTICLE_ENERGY / 2
```

Comment:
Because it's used in TWO places, and in those two places it has to be the same value.
As a rule, if something has to be repeated identically twice i.e. every edit is actually two edits, you should replace it.